includes/definitions/discovery/zyxelusg.yaml

```diff
@@ -0,0 +1,29 @@
+mib: ZYXEL-ES-COMMON
+modules:
+    os:
+        hardware: ZYXEL-ES-COMMON::sysProductModel.0
+        serial: ZYXEL-ES-COMMON::sysProductSerialNumber.0
+        version: ZYXEL-ES-COMMON::sysSwVersionString.0
+    mempools:
+        data:
+            -
+                percent_used: ZYXEL-ES-COMMON::sysMgmtMemUsage
+                index: 0
+            -
+                percent_used: ZYXEL-ES-COMMON::sysMgmtFlashUsage
+                descr: Flash
+                index: 1
+    processors:
+        data:
+            -
+                oid: sysMgmtCPUUsage
```

Comment:
Same format for this please (ZYXEL-ES-COMMON::).

Also, you will need to rebase this as the yaml files now live in: resources/definitions/os_discovery/ and resources/definitions/os_detection/`